rest/model/task.go

```diff
@@ -66,6 +66,7 @@ type APITask struct {
 	TaskGroupMaxHosts       int                 `json:"task_group_max_hosts,omitempty"`
 	Blocked                 bool                `json:"blocked"`
 	Requester               *string             `json:"requester"`
+	ParentPatchID           *string             `json:"parent_patch_id"`
```

Comment:
I was thinking we'd just set the Version field

Reply:
oooh good idea, then we don't need to do it in APITask!